articles/quickstart/webapp/php/01-login.md

```diff
@@ -11,159 +12,214 @@ topics:
 contentType: tutorial
 useCase: quickstart
 github:
-  path: 00-Starter-Seed
+  path: .
 ---
-<%= include('../_includes/_getting_started', { library: 'PHP', callback: 'http://localhost:3000/' }) %>
 
-<%= include('../../../_includes/_logout_url', { returnTo: 'http://localhost:3000' }) %>
+<%= include('../_includes/_getting_started', { library: 'PHP', callback: 'http://127.0.0.1:3000/' }) %>
 
-## Configure PHP to Use Auth0
+<%= include('../../../_includes/_logout_url', { returnTo: 'http://127.0.0.1:3000' }) %>
 
-### Add the Dependencies
+## Integrate your PHP application
 
-::: note
-This sample uses [Composer](https://getcomposer.org/doc/00-intro.md), a tool for dependency management in PHP. It allows you to declare the dependent libraries your project needs and installs them in your project.
-:::
+Let's create a sample application that authenticates a user with a PHP application. We'll take a simple approach here, appropriate for the written format. Still, you should check out the accompanying [Quickstart app on GitHub](https://github.com/auth0-samples/auth0-php-web-app/) for a more robust example.
 
-${snippet(meta.snippets.dependencies)}
+### Installing the PHP SDK
 
-This will create a `vendor` folder and download all the dependencies needed to use the Auth0 PHP SDK. This will also create a `vendor/autoload.php` file which is used in the code samples below to load all necessary classes.
+${snippet(meta.snippets.install)}
 
-### Configure Auth0 PHP SDK
+### Installing HTTP Client and Messaging Factories
```

Comment:
all these paragraphs seem a copy of the ones from the other article. Should they be extracted to a snippet as well?